### PR TITLE
Link to FAQ if Certificate is not available (EXPOSUREAPP-7846)

### PR DESCRIPTION
**Pull request by kolyaopahle**
### Description

This PR adds a new error state described in the following tech spec pr: https://github.com/corona-warn-app/cwa-app-tech-spec/pull/73
 to the test certificate flow for pcr tests

### Steps to reproduce
1. register a negative pcr test with dgc support in an env that does not supply `labId` `bin/cli -e *ENV* pcr --dgc true -r negative`
2. request a certificate for the test
3. check if the certificate fails to fetch in the certificate tab
4. press retry and check if the warning in the screenshot is shown (only german text atm)

### CODE REVIEW

**Comment by BMItter** [comment]
image
tested on oneplus 8 successfully.
code lgtm

**Review by BMItr — APPROVED** [review]

**Comment by BMItter** [comment]
ps: faq link working

**Review by chris-cwa — APPROVED** [review]

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/test/core/TestCertificateRepository.kt [hunk 1]
```diff
@@ -172,12 +175,42 @@ class TestCertificateRepository @Inject constructor(
             }
         }
 
+        // Not sure i really like this
+        internalData.updateBlocking {
```
Comment (BMItter) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/test/core/TestCertificateRepository.kt:179]: @kolyaopahle could you ellaborate your idea a little?

**Pull request merged**